### DISCUSSION TITLE
The conditional check 'custom_registries' failed

### DISCUSSION THREAD

**Discussion by Barlog951** [techno-tim/k3s-ansible#490] [Questions]
Hi all

I was using older version of script from like half year ago ( everything works fine ), now I try new version ( current head )

and this happens

`"msg": "The conditional check 'custom_registries' failed. The error was: error while evaluating conditional (custom_registries): 'custom_registries' is undefined. 'custom_registries' is undefined\n\nThe error appears to be in 'Kubernetes/k3s-ansible/roles/k3s_custom_registries/tasks/main.yml': line 3, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Create directory /etc/rancher/k3s\n  ^ here\n`

servers Ubuntu 22.04 
ansible [core 2.16.5]

I start script from Mac machine

Thanks for help :)

**Comment by timothystewart6** [comment]
can you please post your variables?

**Reply by Barlog951** [reply]
Hi @timothystewart6 here are my vars

**Reply by timothystewart6** [reply]
Hey, can you fill out the template that was prefilled out?  Or paste them here (not in a zip file)
